cloudferrylib/os/identity/keystone.py

```diff
@@ -433,8 +435,7 @@ def update_user(self, user, **kwargs):
 
         return self.keystone_client.users.update(user, **kwargs)
 
-    def get_auth_token_from_user(self):
-        self.keystone_client.authenticate()
```

Comment:
It seems to me that we shouldn't call self.keystone_client.authenticate() each time when we want to get an authentication token. I removed invocation because it overloads keystone by requests. If we want to solve the token expiration problem, we just need to increase the token lifetime.